gnoi_cert/gnoi_cert.go

```diff
@@ -41,7 +42,9 @@ var (
 	certIDs    = flag.String("cert_ids", "", "Comma separated list of Certificate Management certificate IDs for revoke operation")
 	op         = flag.String("op", "get", "Certificate Management operation, one of: provision, install, rotate, get, revoke, check")
 	ca         = flag.String("ca", "", "CA certificate file.")
-	key        = flag.String("key", "", "Private key file.")
+	caKey      = flag.String("ca_key", "", "CA private key file.")
```

Comment:
given the suggested below you wont even need to have any key or certificates requested here. All can be done in credentials.go